demo/api.md

```diff
@@ -78,6 +38,83 @@
 
 </auro-accordion>
 
+#### Dynamic Menu
+
+This example demonstrates loading in dynamic API data to populate the `auro-menu`.
```

Comment:
```suggestion
This example demonstrates a data driven combobox. The data is used to populate the `auro-menu`. In this example, each time input value changes the data is updated and the menu is recreated.

Because the data is filtered by API call using the value in the input field, combobox does not need to handle filtering itself. The `noFilter` attribute on the combobox is required for this example to behave correctly.
```